walkthroughs/tls-with-acm/README.md

```diff
@@ -202,39 +199,42 @@ ColorTellerVirtualNode:
 Additionally, the gateway service will be configured to validate the certificate of the color teller node by specifying the CA that issued it. The spec for the gateway looks like this:
 
 ```yaml
-ColorGatewayVirtualNode:
-  Type: AWS::AppMesh::VirtualNode
+ColorGatewayVirtualGateway:
+  Type: AWS::AppMesh::VirtualGateway
   Properties:
     MeshName: !GetAtt Mesh.MeshName
-    VirtualNodeName: ColorGateway
+    VirtualGatewayName: ColorGateway
     Spec:
-      # BackendDefaults settings are applied to all backends.
       BackendDefaults:
-        # ClientPolicy is a policy for how to handle traffic with backends (i.e. from the client
-        # perspective)
         ClientPolicy:
-          # TLS determines whether or not TLS is negotiated, and how.
           TLS:
-            # Validation determines how to validate a certificate offered by a backend.
             Validation:
-              # Trust is the trust bundle (i.e. set of root certificate authorities) used to validate
-              # the certificate offered by a backend. Certificates signed by one of these certificate
-              # authorities are considered valid.
               Trust:
-                # Use a certificate authority from ACM
                 ACM:
                   CertificateAuthorityArns:
                     - !Ref CertificateAuthorityArn
-      Backends:
-        - VirtualService:
-            VirtualServiceName: !Sub "colorteller.${ServicesDomain}"
       Listeners:
         - PortMapping:
             Port: 80
             Protocol: http
```

Comment:
This port works? Virtual gateways on Fargate needed to be on port >1024. https://docs.aws.amazon.com/app-mesh/latest/userguide/troubleshooting-setup.html#virtual-gateway-low-ports